src/objects.js

```diff
@@ -1,41 +1,54 @@
 const createPerson = (name, age) => {
-  // your code here
+  const obj = { name, age };
+  return obj;
 };
 
 const getName = object => {
-  // your code here
+  return object.name;
 };
 
 const getProperty = (property, object) => {
-  // your code here
+  return object[property];
 };
 
 const hasProperty = (property, object) => {
-  // your code here
+  return property in object;
 };
 
 const isOver65 = person => {
-  // your code here
+  return person.age > 65;
 };
 
 const getAges = people => {
-  // your code here
+  return people.map(object => object.age);
 };
 
 const findByName = (name, people) => {
-  // your code here
+  return people.find(object => object.name === name);
 };
 
 const findHondas = cars => {
-  // your code here
+  return cars.filter(object => {
+    return object.manufacturer === 'Honda';
+  });
 };
 
 const averageAge = people => {
-  // your code here
+  const arr = Object.values(people);
+  const sum = (accumulator, obj) => accumulator + obj.age;
+  const avg = arr.reduce(sum, 0) / arr.length;
+  return avg;
```

Comment:
I see what you were thinking here. A few things you could do though:
- instead of creating the `sum` variable. only to pass it straight into the reduce function you could put it straight in the function itself,
- you dont need to use `Object.values` to create an array to run the reduce function on, you can just call reduce on the array being passed in (`people` is already an array if you look at the test cases),
- the creation of the `avg` variable which is just returned straight afterwards isnt needed, its value can just be returned without storing it in a variable first

Putting all this together, this is the result:
```suggestion
  return (
    people.reduce((accumulator, obj) => accumulator + obj.age, 0) /
    people.length
  );
```